universal-editor-generation-variation/src/aem-universal-editor-1/web-src/src/components/ExtensionRegistration.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * <license header>
+ */
+
+import { Text } from "@adobe/react-spectrum";
+import { register } from "@adobe/uix-guest";
+import { extensionId } from "./Constants";
+
+const generationVariationURL = 'https://experience.adobe.com/#/aem/generate-variations/';
+
+function ExtensionRegistration() {
+  const init = async () => {
+    const guestConnection = await register({
+      id: extensionId,
+      methods: {
+        headerMenu: {
+          getButtons() {
+            return [
+              // @todo YOUR HEADER BUTTONS DECLARATION SHOULD BE HERE
+              {
+                id: 'generate-variations',
+                label: 'Generate Variations',
+                icon: 'OpenIn',
+                onClick() {
+                  window.open(generationVariationURL, '_blank')
```

Comment:
@fe-lix- , Is it enough to redirect a customer to this URL? It looks like some additional info should be provided via query parameters, like `ref`, `repo`, `owner`. Not sure if Universal Editor provides it in the shared context to UI extensions.

Reply:
For me this URL doesn't work, in console output I see
```
Uncaught Error: It seems we're missing the ref, repo or owner search parameter in your application.
    at getWebsiteUrl ...
```

Reply:
Currently the button does not work, Experience Success team they would be able to be "context free" for Summit.